.github/workflows/CI-workflow.yml

```diff
@@ -101,6 +101,11 @@ jobs:
           role-to-assume: ${{ secrets.ML_ROLE }}
           aws-region: us-west-2
 
+      - name: Setup Java ${{ matrix.java }}
```

Comment:
It should fix the compilation problem (but sadly GA does not pick the workflow from the pull request)